tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py312,build_docs_pins
+envlist = py312-pins-pytest_all,build_docs
```

Comment:
These are the environments that get run when we run `tox` without specifying any environments. They needed updating since I recently renamed the environments. We don't actually use this in our CI, so I didn't notice it then.

Reply:
👍